tools/reeve/tests/test_report.py

```
@@ -43,6 +43,27 @@ def test_over_budget_preview_is_flagged():
+        "generatedAt": "2026-08-16T06:00:00Z", "records": [], "previews": [],
+        "reportPlaceholder": False,
+        "runHealth": {
+            "gatheredAt": "2026-08-16T06:00:00Z", "workflows": [],
+            "issues": [{"number": 281, "title": "evil `code`\n# heading [x](y)",
+                        "lockCreatedAt": "2026-08-16T00:00:00Z"}],
+            "openPRs": [], "branches": [],
+        },
+    }
+    cfg = Config()
+    body = render(evaluate(snapshot, cfg), snapshot, cfg)
+    line = next(l for l in body.splitlines() if l.startswith("- #281"))
```

Ruff (0.16.1)

[error] Ambiguous variable name: `l`

(E741)